oauthenticator/tests/test_gitlab.py

```diff
@@ -28,6 +28,7 @@ def user_model(username):
     return {
         "username": username,
         "id": id,
+        "groups": ["group1"],
```

Comment:
For now i'm happy to settle with this, but i think its worth looking up how this can be realistic for all authenticator classes responses - requires some effort though

Reply:
I opened https://github.com/jupyterhub/oauthenticator/issues/759 to reflect this